packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -123,6 +141,15 @@
 			path = TestImages;
 			sourceTree = "<group>";
 		};
+		6801C8372555D726009DAF8D /* RunnerUITests */ = {
```

Comment:
This is describing the files that are in the `RunnerUITests` group in Xcode, so the files/folders explorer thing on the left side.